src/cider/nrepl/middleware/debug.clj

```diff
@@ -117,12 +117,19 @@
   "Bound by the `breakpoint` macro to the local &env."
   {})
 
+(defn pr-short
+  "Like `pr-str` but limited in length and depth."
+  [x]
+  (binding [*print-length* 6
```

Comment:
Isn't it better to give some control to the users to configure this? E.g. we can pass some data from cider. Or maybe this is just going to be overkill...